docker-compose.yml

```
@@ -26,8 +26,8 @@ services:
       dockerfile: ./Dockerfile
       target: runtime
     ports:
-      - "8000:8000"
-      - "26466:26466"
+      - "${VULN_HOST_HTTP_PORT:-8000}:8000"
+      - "${VULN_HOST_INPUT_PORT:-26466}:26466"
```

Thoughts on these variable names? Change from `VULN_HOST` (which could sound like "vulnerable host") to `WORKFLOW`, and make the port functionality more specific.
```suggestion
      - "${WORKFLOW_HOST_JUPYTER_PORT:-8000}:8000"
      - "${WORKFLOW_HOST_SCAN_PORT:-26466}:26466"
```